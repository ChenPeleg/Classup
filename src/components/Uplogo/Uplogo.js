import React from "react";
import styles from "./uplogo.module.scss";

const uplogo = (props) => {
  return (
    <div className={styles.logo + " " + styles.LogoInHeader}>Class-Up!</div>
  );
};
export default uplogo;
