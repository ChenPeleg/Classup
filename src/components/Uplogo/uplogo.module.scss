@font-face {
  font-family: "LuckiestGuy";
  font-weight: 400;
  font-style: normal;
  src: url("LuckiestGuy-Regular.ttf") format("truetype");
}

.logo {
  display: inline-block;
  color: rgb(10, 59, 110);

  font-family: LuckiestGuy;
  text-shadow: 1px 1px rgb(151, 88, 82), 2px 2px rgba(205, 245, 94, 0.836),
    3px 3px rgba(122, 94, 245, 0.836), 4px 4px rgba(110, 18, 95, 0.836),
    5px 5px rgba(205, 245, 94, 0.836);
}

.LogoInHeader {
  vertical-align: middle;
  font-size: 32px;
  padding-bottom: 5px;
}
