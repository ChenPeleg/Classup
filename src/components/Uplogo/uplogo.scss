@font-face {
  font-family: "LuckiestGuy";
  font-weight: 400;
  font-style: normal;
  src: url("LuckiestGuy-Regular.ttf") format("truetype");
}

.logo {
  color: rgb(10, 59, 110);
  font-size: 42px;
  padding: 10px;
  font-family: LuckiestGuy;
  text-shadow: 1px 1px rgb(151, 88, 82), 2px 2px rgba(205, 245, 94, 0.836),
    3px 3px rgba(122, 94, 245, 0.836), 4px 4px rgba(110, 18, 95, 0.836),
    5px 5px rgba(205, 245, 94, 0.836);
}
