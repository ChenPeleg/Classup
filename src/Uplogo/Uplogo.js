import React from "react";
import "./uplogo.css";

const uplogo = props => {
  return <div className="logo">Class-Up!</div>;
};
export default uplogo;
