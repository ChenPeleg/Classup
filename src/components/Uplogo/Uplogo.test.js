import React from "react";
import { shallow } from "enzyme";
import Uplogo from "./Uplogo";
const wrap = shallow(<Uplogo />)

describe("<Uplogo>", () => {
  it("renders correctly", () => { expect(wrap).toMatchSnapshot() })
})