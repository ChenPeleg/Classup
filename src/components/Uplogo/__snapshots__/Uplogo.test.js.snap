// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Uplogo> renders correctly 1`] = `
<div
  className="logo LogoInHeader "
>
  Class-Up!
</div>
`;
